GAME_DESIGN_PRINCIPLES.md

```diff
@@ -1,57 +1,52 @@
 
 ### No Teleportation
-To immerse the player and let them feel the sense of scale the world of Cubyz has, teleportation is not allowed, as the player will see less of the world that way.
+Teleportation makes the game less immersive, as it diminishes the exploring aspect and doesn't let the player get a good sense of the scale of the world.
 
 ### No Automation
-Instead of staying in one place and farming everything, the player should have to take advantage of the infinite nature of the world to gain resources, meaning that a majority of resources are non-renewable.
+Having quick, infinite resources at the palm of players' hands discourages exploration, as the player will never need to forage or search for blocks they desire.
 
-### Mobs Don't Respawn
-To prevent farming, mobs will not respawn naturally once killed. This will also make the player more aware that killing has long-term consequences in the area it occurs in.
+### Mobs Don't Respawn Naturally
+Clearing a dangerous area of its monsters will make it safe to build and explore.
+This rule is also here to prevent mob farming.
 
 ### No Passive Animals
-Instead of animals standing by to let you kill them, they will run from the player, attack back, and use defense mechanisms. This should make hunting and taming much more engaging.
+Animals do not want to die, so they will either run from an attacking player or attempt to defend themselves. The player will have to strategize to be able to hunt.
 
 ### No Unbreakable Tools
-In Minecraft, a tool is an investment you can make. You can enchant the tool, rename it, make it your own; there's an incentive to make it last forever with the use of mending. In Cubyz though, you are not creating an investment, but rather a tool to use until it breaks. If a player gets too attached to their tool, then they won't want to make other types of tools, and lower-tier materials will see less use as they will never be used in tool-making.
+If a player gets too attached to their amazing tool, they won't want to make other types of tools, and low-tier materials will see less use as they will never be used in tool-making.
 
 ### Avoid Clutter
-Inventories will often fill up with random items that the player does not want, making their inventory hard to manage. To mitigate this, try finding uses for existing items before adding new ones, and find ways to prevent items from finding their way into a player's inventory when they don't want it.
+Inventories will often fill up with random items that the player does not want, making their inventory hard to manage. To mitigate this, find uses for existing items before adding new ones and prevent items from finding their way into a player's inventory when they don't want it.
+
+### Use Realism Tastefully
+Cubyz is not a realistic game. The world is very whimsical and has rules of its own, sometimes inspired by the real world, but never copying it outright.
```

Comment:
I think sometimes it might make sense to copy real world mechanics, but there should be a good reason for it.
```suggestion
Cubyz is not a realistic game. The world is very whimsical and has rules of its own, sometimes inspired by the real world, but never copied without a good reason.
```